### ISSUE TITLE
Unable to run Wyze Bridge + Frigate on Same Server Due to Overlapping Ports

### ISSUE THREAD

**Issue by diggity801** [mrlt8/docker-wyze-bridge#1003]
Greetings!

I want to run the Wyze bridge on the same server via Docker Compose as I do Frigate. The problem is that both projects decided that port 5000 would be a good choice. When I expose different ports, it essentially breaks the Wyze Bridge application. Is there any way around this? I'm assuming that most people using this bridge are using Frigate, too, so I'm not sure why port 5000 was chosen.

Very best,
David

**Comment by mrlt8** [comment]
Good point. 5000 is the just default port for flask and that's why you'll see many projects the use flask using the default 5000.
Open to suggestions for alternate port numbers.

**Comment by diggity801** [comment]
I actually got it to work aftering trying to expose different ports again
in Docker Compose. I am using 5001 for the GUI and it works good for me
with Frigate. I incremented the ports by 1. I do think it would be pretty
sweet not to have to work around that though and I encourage you to change
the port numbers to numbers that Frigate does not use.

Thank you sir and appreciate your time!

David Lloyd
